pkg/controller/migmigration/backup.go

```diff
@@ -281,6 +282,52 @@ func (t *Task) updateBackup(backup *velero.Backup) error {
 	return nil
 }
 
+func (t *Task) deleteBackups() error {
+	client, err := t.getSourceClient()
+	if err != nil {
+		log.Trace(err)
+		return err
+	}
+
+	labels := t.Owner.GetCorrelationLabels()
+	labels["migmigration"] = string(t.Owner.GetUID())
```

Comment:
This should already be included in the correlation labels.

Reply:
Thanks, I didn't realize that :(